### ISSUE TITLE
Can't install spacemacs and M is undefined

### ISSUE THREAD

**Issue by lassiter** [syl20bnr/spacemacs#12598]
Getting this error from clean install:
```
--> installing package: paradox@spacemacs-ui... [71/90]
Package spinner is unavailable. Is the package name misspelled?

An error occurred while installing paradox (error: (error Package ‘spinner-1.7.3’ is unavailable))
--> installing package: undo-tree@spacemacs-editing... [83/90]
Package undo-tree is unavailable. Is the package name misspelled?
--> installing package: ws-butler@spacemacs-editing... [90/90]
```

Tried following instructions here to install: https://elpa.gnu.org/packages/undo-tree.html

M is undefined in the editor so can't run `M-x package-install RET undo-tree RET`

**Comment by tuxayo** [comment]
Thanks I was just blocked by that :)

So even for the long run, the solution will be to use `emacs --insecure` ?

**Comment by duianto** [comment]
This issue has a workaround that seems to fix installation issues:
Failed to download 'gnu' archive. Bad request. (Solution included) #12535
